From e9e051f50f5c4e324bc37b9a67996ca2df1530f5 Mon Sep 17 00:00:00 2001
Message-Id: <e9e051f50f5c4e324bc37b9a67996ca2df1530f5.1645476282.git.github@sicherha.de>
From: Christoph Erhardt <github@sicherha.de>
Date: Mon, 21 Feb 2022 21:44:34 +0100
Subject: [PATCH] Skip reloc-rodata test on aarch64

The test expects an error when an object compiled with `-fno-PIC` is
linked with `-pie`, but that doesn't happen on aarch64.

Signed-off-by: Christoph Erhardt <github@sicherha.de>
---
 test/elf/reloc-rodata.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/elf/reloc-rodata.sh b/test/elf/reloc-rodata.sh
index 2eeb237c..b9cf8161 100755
--- a/test/elf/reloc-rodata.sh
+++ b/test/elf/reloc-rodata.sh
@@ -10,6 +10,8 @@ mold="$(pwd)/mold"
 t=out/test/elf/$testname
 mkdir -p $t
 
+[ "$(uname -m)" = aarch64 ] && { echo skipped; exit; }
+
 cat <<EOF | $CC -fno-PIC -c -o $t/a.o -xc -
 #include <stdio.h>
 
-- 
2.35.1

